extensions/data-plane/data-plane-google-storage/README.md

```diff
@@ -5,26 +5,35 @@
 This module contains a Data Plane extension to copy data to and from Google cloud storage.
 
 ### Authentication
+Google storage data plane supports three different approaches for authentication:
+* Default authentication:
+    * Authenticates against the Google Cloud API using the [Application Default Credentials](https://cloud.google.com/docs/authentication#adc).
+    * These will automatically be provided if the connector is deployed with the correct service account attached.
+* Service Account key file
+    * Authentication using a Service Account key file exported from Google Cloud Platform
+    * Service Account key file can be stored in a vault or encoded as base64 and provided in the dataAddress.
```

Comment:
Pls check [GcpConfiguration.java](https://github.com/eclipse-edc/Technology-Gcp/blob/17be30b16d1688c563e1a66ec22ea87e09bf6c8a/extensions/common/gcp/gcp-core/src/main/java/org/eclipse/edc/gcp/common/GcpConfiguration.java#L31) , as this provide a configuration at connector level where service account is optionally set.